conans/client/rest/rest_client.py

```diff
@@ -367,19 +374,24 @@ def _post_json(self, url, payload):
         return response
 
     def _get_json(self, url, data=None):
+        t1 = time.time()
+        headers = self.custom_headers
         if data:  # POST request
-            headers = {'Content-type': 'application/json',
-                       'Accept': 'text/plain',
-                       'Accept': 'application/json'}
-            headers.update(self.custom_headers)
+            headers.update({'Content-type': 'application/json',
+                            'Accept': 'text/plain',
+                            'Accept': 'application/json'})
             response = self.requester.post(url, auth=self.auth, headers=headers,
                                            verify=self.verify_ssl,
                                            stream=True,
                                            data=json.dumps(data))
         else:
-            response = self.requester.get(url, auth=self.auth, headers=self.custom_headers,
+            response = self.requester.get(url, auth=self.auth, headers=headers,
                                           verify=self.verify_ssl,
                                           stream=True)
+
+        duration = time.time() - t1
+        method = "POST" if data else "GET"
+        log_client_rest_api_call(url, method, duration, headers)
```

Comment:
If these headers will carry basic-auth to get JWT, please remove that header from logging